### PR TITLE
RC: Make sure "connect to database" and "Default user" pages are up-to-date

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-4392

### CODE REVIEW

File: content/operate/rc/databases/connect.md [hunk 1]
```diff
@@ -55,10 +55,10 @@ A Redis client is a software library or tool that enables applications to intera
 
 The connection wizard provides code snippets to connect to your database with the following programming languages:
 
-- node.js using [node-redis](https://github.com/redis/node-redis/blob/master/README.md)
+- node.js using [node-redis]({{< relref "/develop/connect/clients/nodejs" >}})
 - .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
-- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
-- Java using [Jedis](https://github.com/redis/jedis#jedis)
+- Python using [redis-py]({{< relref "/develop/connect/clients/python/redis-py" >}})
+- Java using [Jedis]({{< relref "/develop/connect/clients/java/jedis" >}})
```
Comment (dwdougherty) [inline content/operate/rc/databases/connect.md:61]: Why not this link for C#/.Net ?

https://redis.io/docs/staging/DOC-4392/develop/connect/clients/dotnet/

File: content/operate/rc/rc-quickstart.md [hunk 2]
```diff
@@ -115,10 +115,10 @@ A Redis client is a software library or tool that enables applications to intera
 
 The connection wizard provides code snippets to connect to your database with the following programming languages:
 
-- node.js using [node-redis](https://github.com/redis/node-redis/blob/master/README.md)
+- node.js using [node-redis]({{< relref "/develop/connect/clients/nodejs" >}})
 - .NET using [StackExchange.Redis](https://stackexchange.github.io/StackExchange.Redis/)
-- Python using [redis-py](https://github.com/redis/redis-py#redis-py)
-- Java using [Jedis](https://github.com/redis/jedis#jedis)
+- Python using [redis-py]({{< relref "/develop/connect/clients/python/redis-py" >}})
+- Java using [Jedis]({{< relref "/develop/connect/clients/java/jedis" >}})
```
Comment (dwdougherty) [inline content/operate/rc/rc-quickstart.md:121]: Same comment as above.

**Review by dwdougherty — COMMENTED** [review]
See comments re. C#/.Net. StackExchange.Redis is owned by Redis, but NRedisStack is.

File: content/operate/rc/databases/connect.md (hunk 1 above)
Comment (cmilesb) [reply content/operate/rc/databases/connect.md:61]: This is just listing the examples in the connection wizard, not the clients that Redis owns. @andy-stark-redis is working with the FE infrastructure team to change the examples in the connection wizard to those that are owned by Redis, but until that is live on production, the C#/.NET example is in StackExchange.Redis and not NRedisStack.

Comment (dwdougherty) [reply content/operate/rc/databases/connect.md:61]: Got it. Apologies for intruding.

Comment (cmilesb) [reply content/operate/rc/databases/connect.md:61]: All good, it makes sense.

**Review by dwdougherty — APPROVED** [review]

**Pull request merged**